vm86/msdos.cpp

```diff
@@ -1536,6 +1536,7 @@ extern "C"
                     save_context(&context);
                     call_native_wndproc_context(&context);
                     load_context(&context);
+                    m_eip = context.Eip;
```

Comment:
It is already set in load_context.
https://github.com/otya128/winevdm/blob/bfc5ae695bf4f0fcc4d488573ca48130eab0628f/vm86/msdos.cpp#L670

Reply:
It was fixed in https://github.com/otya128/winevdm/commit/52cbec439b1f35d39a42a5f5f05b2cfd85842d8b and I wasn't updated.